testing/tests/ui/default.stderr

```diff
@@ -0,0 +1,88 @@
+error: the `default` filter requires a variable name on its left-hand side
+ --> DefaultLhsExpr.html:1:3
+       "(1 + 1) | default(2) }}"
+ --> tests/ui/default.rs:6:35
+  |
+6 | #[template(ext = "html", source = "{{ (1 + 1) | default(2) }}")]
+  |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: `default_value` argument is missing when calling `default` filter; its arguments are: (default_value, [boolean])
+ --> DefaultNoDefaultValue.html:1:3
+       "var | default }}"
+  --> tests/ui/default.rs:10:35
+   |
+10 | #[template(ext = "html", source = "{{ var | default }}")]
```

Comment:
This actually suggests a bit of a jinja / rust impedance mismatch here. In Rust, we have a few different operations like `map_or_default` and `unwrap_or_default`. Intuitively, `var | default` when the type of var implements the `Default` trait seems like it would be obvious that it should return default. I know that a user could call `.unwrap_or_default()` or whatever there, but I'm curious whether it might be worth extending this to allow the obvious intuitive thing there. (I'm not fully certain of this, either way, WDYT?)